cli/internal/hashing/package_deps_hash.go

```diff
@@ -50,7 +50,20 @@ func GetPackageDeps(rootPath turbopath.AbsolutePath, p *PackageDepsOptions) (map
 		// Note this package.json will be resolved relative to the pkgPath.
 		calculatedInputs = append(calculatedInputs, "package.json")
 
-		absoluteFilesToHash, err := globby.GlobFiles(pkgPath.ToStringDuringMigration(), calculatedInputs, nil)
+		// The input patterns are relative to the package.
+		// However, we need to change the globbing to be relative to the repo root.
+		// Prepend the package path to each of the input patterns.
+		prefixedInputPatterns := make([]string, len(calculatedInputs))
+		for index, pattern := range calculatedInputs {
+			rerooted, err := rootPath.PathTo(pkgPath.Join(pattern))
+			if err != nil {
+				return nil, err
+			}
+			prefixedInputPatterns[index] = rerooted
+		}
```

Comment:
If `calculatedInputs` is already a copy made within this function, could we avoid allocating again for `prefixedInputPatterns` here and just mutate `calculatedInputs`?

Reply:
We're using the _unprefixed_ version in a different location just a little bit later. There is some alignment to do here, but I'm planning to delay that until we get to refactoring all of our globbing code in a single pass. (At which point I'm _imagining_ it will become aware of "repo root" and "package root" concepts, making this irrelevant.)

Reply:
(And, allocation-wise, `inputs` are human-authored so there is a reasonable bound on how much gets allocated here that I'm comfortable with trading for correctness.)

Reply:
SGTM then. I guess the input patterns tend to be pretty short too, so it's not like it's an expensive allocation here. 🤔